sim/build.sbt

```diff
@@ -47,6 +50,12 @@ lazy val testchipip    = ProjectRef(chipyardDir, "testchipip")
 lazy val sifive_blocks = ProjectRef(chipyardDir, "sifive_blocks")
 lazy val firechip      = ProjectRef(chipyardDir, "firechip")
 
+lazy val chiselLib = "edu.berkeley.cs" %% "chisel3" % chiselVersion
```

Comment:
Why do you have `chiselLib` here if you're not using it?

Reply:
Agreed. There is no point in using this if we aren't using SBT-Sriracha in FireSim.